build.sbt

```diff
@@ -19,11 +19,24 @@ lazy val core = module("core")
   .dependsOn(meta)
   .settings(libraryDependencies ++= Seq(
     "org.typelevel" %%% "cats-core"   % "1.1.0",
-    "org.typelevel" %%% "cats-free"   % "1.1.0"))
+    "org.typelevel" %%% "cats-free"   % "1.1.0",
+    "org.typelevel" %%% "kittens"     % "1.1.0",
+    compilerPlugin("org.scalamacros" % "paradise" % "2.1.0" cross CrossVersion.patch)
+  ))
 
 lazy val coreJVM = core.jvm
 lazy val coreJS  = core.js
 
+lazy val macros = module("macros")
```

Comment:
Let's add the macros project to the aggregate list at the top on the root project